doc/reference/reference_rock/vshard/vshard-router.rst

```diff
@@ -345,6 +347,27 @@ Router public API
         If the function returns ``nil`` or ``box.NULL`` from one of the storages,
         it will not be present in the resulting map.
 
+        If the ``return_raw`` option is used,
+        the result is a map of the following format: ``{[replicaset_uuid] = msgpack.object}``
+        where ``msgpack.object`` is an object that stores a MessagePack array with the results returned from the storage map function.
+
+        The option use case is the same as in using ``net.box``: to avoid decoding of the call results into Lua.
+        The option can be helpful if a router is used as a proxy and results received from a storage is big.
```

Comment:
`results received from a storage is big` -> `results received from storages are big`.

Reply:
Corrected.